ee-feature-pack/galleon-shared/src/main/resources/content/bin/appclient.bat

```diff
@@ -83,10 +83,12 @@ if "x%JAVA_HOME%" == "x" (
 )
 
 rem set default modular jvm parameters
-setlocal EnableDelayedExpansion
-call "!DIRNAME!common.bat" :setDefaultModularJvmOptions "!JAVA_OPTS!"
-set "JAVA_OPTS=!JAVA_OPTS! !DEFAULT_MODULAR_JVM_OPTIONS!"
-setlocal DisableDelayedExpansion
+if "%SECMGR%" == "true" (
+    setlocal EnableDelayedExpansion
+    call "!DIRNAME!common.bat" :setDefaultModularJvmOptions "!JAVA_OPTS!"
+    set "JAVA_OPTS=!JAVA_OPTS! !DEFAULT_MODULAR_JVM_OPTIONS!"
+    setlocal DisableDelayedExpansion
+)
```

Comment:
This is not about SM. It's about --add-opens etc and should not be changed as part of this JIRA.  If something needs changing it's another JIRA.

Reply:
Good catch on all these. I clearly scanned for the wrong function name.